content/admin/convert-account.md

```diff
@@ -6,9 +6,14 @@ aliases:
 - /docker-hub/convert-account/
 ---
 
+> **Tip**
+>
+> You can convert an account with a Docker Personal subscription to an organization. If you need to convert an account with a Docker Pro subscription, [contact support](https://hub.docker.com/support/contact).
```

Comment:
**suggestion:** I believe in other places, we use the term `your personal account` when referring to a Docker user account.

```suggestion
> You can convert your personal account with a Docker Personal subscription to an organization. If you need to convert your personal account with a Docker Pro subscription, [contact support](https://hub.docker.com/support/contact).
```